.github/actions/test_ios_rntester/action.yml

```diff
@@ -0,0 +1,158 @@
+name: test_ios_rntester
+description: Test iOS RNTester
+inputs:
+  jsengine:
+    description: Which JavaScript engine to use. Must be one of "Hermes", "JSC".
+    default: Hermes
+  use-frameworks:
+      description: The dependency building and linking strategy to use. Must be one of "StaticLibraries", "DynamicFrameworks"
+      default: StaticLibraries
+  architecture:
+    description: The React Native architecture to Test. RNTester has always Fabric enabled, but we want to run integration test with the old arch setup
+    default: NewArch
+  ruby-version:
+    description: The version of ruby that must be used
+    default: 2.6.10
+  run-unit-tests:
+    description: whether unit tests should run or not.
+    default: false
+  hermes-tarball-artifacts-dir:
+    description: The directory where the hermes tarball artifacts are stored
+    default: /tmp/hermes/hermes-runtime-darwin
+  flavor:
+    description: The flavor of the build. Must be one of "Debug", "Release".
+    default: Debug
+  hermes-version:
+    description: The version of hermes
+    required: true
+  react-native-version:
+    description: The version of react-native
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Setup xcode
+      uses: ./.github/actions/setup-xcode
+    - name: Setup node.js
+      uses: ./.github/actions/setup-node
+    - name: Run yarn
+      shell: bash
+      run: yarn install --non-interactive
+    - name: Cache setup
+      id: cache_setup
+      uses: ./.github/actions/cache_setup
+      with:
+        hermes-version: ${{ inputs.hermes-version }}
+        react-native-version: ${{ inputs.react-native-version }}
+    - name: Delete iOS Simulators
+      shell: bash
+      run: sudo rm -rf /Library/Developer/CoreSimulator/Profiles/Runtimes/ &
```

Comment:
this can probably go. Can we try to remove this and see what happens?

Reply:
Because [this note](https://github.com/facebook/react-native/blob/401f2fbb14b1cfea93220810cce79b47720ecc72/.circleci/configurations/jobs.yml#L536), I'm guessing if we remove this, the job fails because we run out of disk space. I'll try to remove this and see what happens.

Reply:
Seems to work without this.